tests/Oryx.RuntimeImage.Tests/DotNetCore/DotNetCoreRuntimeImageTestBase.cs

```diff
@@ -0,0 +1,26 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.Oryx.Tests.Common;
+using System.IO;
+using System.Net.Http;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.RuntimeImage.Tests
+{
+    public class DotNetCoreRuntimeImageTestBase : TestBase, IClassFixture<TestTempDirTestFixture>
```

Comment:
Looks that this file is not being used, should we remove it or include it on the top of the dotnet runtimeImageTests